src/pybind/CMakeLists.txt

```diff
@@ -68,27 +63,20 @@ if(NMODL_ENABLE_PYTHON_BINDINGS)
   if(LINK_AGAINST_PYTHON)
     target_link_libraries(_nmodl PRIVATE pywrapper)
   endif()
-
-  add_custom_command(
-    OUTPUT ${PROJECT_BINARY_DIR}/lib/nmodl
-    COMMAND ${CMAKE_COMMAND} -E copy_if_different $<TARGET_FILE:_nmodl> ${PROJECT_BINARY_DIR}/nmodl
-    DEPENDS ${NMODL_PYTHON_FILES_IN} _nmodl
-    COMMENT "-- COPYING NMODL PYTHON MODULE --")
 endif()
 
-add_custom_target(copy_python_files ALL DEPENDS ${NMODL_PYTHON_FILES_OUT})
-add_custom_command(
-  OUTPUT ${NMODL_PYTHON_FILES_OUT}
-  COMMAND ${CMAKE_COMMAND} -E copy_directory ${NMODL_PROJECT_SOURCE_DIR}/nmodl
-          ${PROJECT_BINARY_DIR}/lib/nmodl
-  DEPENDS ${NMODL_PYTHON_FILES_IN}
-  COMMENT "-- COPYING NMODL PYTHON FILES --")
-
 # =============================================================================
 # Copy python binding components and examples into build directory
 # =============================================================================
-file(GLOB NMODL_PYTHON_HELPER_FILES "${NMODL_PROJECT_SOURCE_DIR}/nmodl/*.py")
-file(COPY ${NMODL_PYTHON_HELPER_FILES} DESTINATION ${CMAKE_BINARY_DIR}/lib/nmodl/)
+file(
+  GLOB NMODL_PYTHON_FILES
+  RELATIVE "${NMODL_PROJECT_SOURCE_DIR}/nmodl/"
+  CONFIGURE_DEPENDS "${NMODL_PROJECT_SOURCE_DIR}/nmodl/*.py")
```

Comment:
Following the previous use of `file(GLOB` in NMODL.

We can discuss "hard-coding" the list of files. There's a similar `GLOB` for creating a test for each mod file.